### PR TITLE
Init Azure.ResourceManager.DesktopVirtualization

### PR DESCRIPTION
**Pull request by ArthurMa1978**
Here is the [APIView](https://apiview.dev/Assemblies/Review/8b86340b0ad7420ca54d4942be9045d9)

### CODE REVIEW

File: sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/Generated/Application.cs [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Core;
+using Azure.ResourceManager.DesktopVirtualization.Models;
+
+namespace Azure.ResourceManager.DesktopVirtualization
+{
+    /// <summary> A Class representing a Application along with the instance operations that can be performed on it. </summary>
+    public partial class Application : ArmResource
```
Comment (fengzhou-msft) [inline sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/Generated/Application.cs:22]: Avoid using single-word class name.

**Comment by fengzhou-msft** [comment]
api file is missing.

File: sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/Generated/Models/MsixImageURI.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+
+namespace Azure.ResourceManager.DesktopVirtualization.Models
+{
+    /// <summary> Represents URI referring to MSIX Image. </summary>
+    public partial class MsixImageURI
```
Comment (Yao725) [inline sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/Generated/Models/MsixImageURI.cs:13]: Should be Uri, we can use the renaming rule to update this:
```yaml
rename-rules:
  URI: Uri
```

File: sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/Generated/Models/SSOSecretType.cs [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.DesktopVirtualization.Models
+{
+    /// <summary> The type of single sign on Secret Type. </summary>
+    public readonly partial struct SSOSecretType : IEquatable<SSOSecretType>
```
Comment (Yao725) [inline sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/Generated/Models/SSOSecretType.cs:14]: Should be `Sso`?

**Review by Yao725 — COMMENTED** [review]
Can we have a apiview of this RP to make the review easier?

File: sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/autorest.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+azure-arm: true
+csharp: true
+library-name: DesktopVirtualization
+namespace: Azure.ResourceManager.DesktopVirtualization
+require: https://github.com/Azure/azure-rest-api-specs/blob/49af362e33d89967d7776fdd3a26d5462c9fbb59/specification/desktopvirtualization/resource-manager/readme.md
+tag: package-2021-07
+output-folder: Generated/
+clear-output-folder: true
+mgmt-debug:
+  show-request-path: true
```
Comment (ArcturusZhang) [inline sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/autorest.md:16]: We no longer need this to show the request paths, now it shows the request paths by default
```suggestion
```

File: .vscode/cspell.json [hunk 5]
```diff
@@ -152,6 +152,15 @@
...
         "vSphere"
       ]
+    },
+	{
+      "filename": "**/sdk/desktopvirtualization/**/*.cs",
+      "words": [
+        "logix",
+        "msix",
+        "ssoadfs",
+        "unhide"
```
Comment (ArcturusZhang) [inline .vscode/cspell.json:162]: Are these words abbreviations in the SDK? Maybe we need to add them to the `rename-rules` if they are

File: sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/autorest.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+azure-arm: true
+csharp: true
+library-name: DesktopVirtualization
+namespace: Azure.ResourceManager.DesktopVirtualization
+require: https://github.com/Azure/azure-rest-api-specs/blob/49af362e33d89967d7776fdd3a26d5462c9fbb59/specification/desktopvirtualization/resource-manager/readme.md
+tag: package-2021-07
+output-folder: Generated/
+clear-output-folder: true
+mgmt-debug:
```
Comment (ArcturusZhang) [inline sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/autorest.md:15]: Could we also add a `rename-rules` section here to ensure the abbreviation casing is consistent between RPs?

File: sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/Generated/Application.cs (hunk 1 above)
Comment (ArthurMa1978) [reply sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/Generated/Application.cs]: fixed

File: sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/Generated/Models/MsixImageURI.cs (hunk 2 above)
Comment (ArthurMa1978) [reply sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/Generated/Models/MsixImageURI.cs]: the rename-rules can fix the model name but not work for the parameter name, @ArcturusZhang  will check it later

File: .vscode/cspell.json (hunk 5 above)
Comment (ArthurMa1978) [reply .vscode/cspell.json]: except the 'unhide' others are all common names of some industry product / standard

File: sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/autorest.md (hunk 6 above)
Comment (ArthurMa1978) [reply sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/autorest.md]: done

**Review by Yao725 — APPROVED** [review]

File: sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/Generated/Models/MsixImageURI.cs (hunk 2 above)
Comment (ArthurMa1978) [reply sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/Generated/Models/MsixImageURI.cs]: let's checkin first, then first this naming later once the codegen fixed

**Pull request merged**